content/en/methods/accounts.md

```diff
@@ -1872,15 +1994,16 @@ Can sometimes be returned if the account already endorsed.
 ## Unfeature account from profile {#unpin}
 
 ```http
-POST /api/v1/accounts/:id/unpin HTTP/1.1
```

Comment:
Maybe we should still reference `unpin` somewhere and say it has been renamed/deprecated? “renamed to `unendorse`” does not tell what it was renamed from.

Reply:
Changed to "renamed `pin` to `endorse`". I'm not sure if there is a format we use for such renames and aliases...

Reply:
What about keeping the old `pin` entry and marking it as deprecated, and adding a new one for `endorse`?

Reply:
agreed with @renchap -- this is technically a new endpoint because it has a different URI.